crates/recursor/src/common.rs

```diff
@@ -0,0 +1,149 @@
+use std::time::Instant;
```

Comment:
Not a big fan of files that are this small. Maybe inline the contents of `common` and `dnssec_policy` into `lib.rs`?

Also please keep moving code around in a separate commit.